docs/example-scenario/infrastructure/multi-tier-app-disaster-recovery.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Highly available (HA) and disaster recovery (DR) configured multi-tier web application
```

Comment:
Title should be easier to parse.  Let's lead with the app and drop the abbreviations.

How about: Multi-tier web application built for high availability and disaster recovery

Reply:
liked the title...changed it